eisbuk-admin/src/components/atoms/CustomerList/CustomerList.tsx

```diff
@@ -48,9 +48,11 @@ const CustomerList: React.FC<Props> = ({
   const searchRegex = new RegExp(searchString, "i");
 
   return (
-    <div className={className}>
+    <div className={classNames?.rootDiv}>
```

Comment:
Generally, this `?` is unnecessary as you have a fallback, so `classNames` will always be defined. The problem could arise if someone passes `classNames={ }`, overriding your fallback, but `classNames.rootDiv` will be `undefined` (and it should be `string`). In that case, this will still not fix it and is one of the reasons for my comment above: when you pass two (optional) strings, instead of an object, it's easier (and more straightforward) to provide a fallback, catching all of the cases.